.circleci/config.yml

```diff
@@ -126,6 +126,7 @@ jobs:
     - run:
         name: "Run unit tests"
         command: |
+          export CI=true
```

Comment:
This is already set by circleci and most CI systems .e.g https://app.circleci.com/pipelines/github/Kong/kong-mesh/5059/workflows/1becd198-93fa-4cc0-897b-b95e02cd460c/jobs/37550?invite=true#step-99-3